packages/base/src/widget.ts

```diff
@@ -524,6 +524,8 @@ export class WidgetModel extends Backbone.Model {
    * binary array buffers.
    */
   serialize(state: Dict<any>): JSONObject {
+    const deepcopy = 
+      globalThis.structuredClone || (x: any) => JSON.parse(JSON.stringify(x));
```

Comment:
From the test failures, it looks like we need parentheses around the fallback function.

Reply:
Good catch, thanks.